catalog/transformer/catalog-transformer-rtf/pom.xml

```diff
@@ -0,0 +1,190 @@
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-dateTime.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-dateTime</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-location.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-location</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-media.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-media</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-security.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-security</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-topic.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-topic</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-validation.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-validation</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>src/main/resources/default-category-version.config</file>
+                                    <type>config</type>
+                                    <classifier>default-category-version</classifier>
+                                </artifact>
+                            </artifacts>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Export-Package>
+                            ddf.catalog.transformer.output.rtf,
```

Comment:
Appears so, the bundle doesn't want to start without the service implementation package being visible.

Reply:
That's strange. Exports should not affect the bundle's own ability to start. It could affect other bundles though. We should figure out what's going and why before merging. If we need to export an implementation class for things to work we may need to split the bundle.